### ISSUE TITLE
Making this persistent and with memory

### ISSUE THREAD

**Issue by abdcef** [davila7/file-gpt#4]
Is it possible to make this persistent so that I do not have to repeat the embeddings?
Also to have longer conversations by the bot retaining the knowledge of previous queries

**Comment by davila7** [comment]
Yes, it's possible, you should save the embedding to a vector database like Pinecone.

For the chat to have long-term memory you can use this Langchain function
https://langchain.readthedocs.io/en/latest/modules/memory.html

If you can make progress on this you can do a PR and I'll add it to the repo.

**Comment by abdcef** [comment]
Thanks. Yes, trying to explore in that direction.
